taichi/transforms/lower_ast.cpp

```diff
@@ -248,7 +258,7 @@ class LowerAST : public IRVisitor {
       auto &&new_for = std::make_unique<RangeForStmt>(
           begin, end, std::move(stmt->body), stmt->is_bit_vectorized,
           stmt->num_cpu_threads, stmt->block_dim, stmt->strictly_serialized,
-          /*range_hint=*/fmt::format("arg {}", tensor->arg_id));
+          /*range_hint=*/fmt::format("arg {}", arg_id));
```

Comment:
Is `arg_id` not given the correct value?

Reply:
Hmm I mean you need to assign the value to `arg_id` so that `RangeForStmt` can get the correct hint string.

Reply:
ah, i forgot about it lol